app/views/users/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+
+<html>
+    <h1>Sign up</h1>
+
+    <!--form action="/users" method="post" accept-charset="UTF8">
+        <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+        <label for="username">Username</label>
+        <input name="user[username]" type="text" />
+        <label for="email">Email</label>
+        <input name="user[email]" type="email" />
+        <label for="password">Password</label>
+        <input name="user[password]" type="password" />
+        <input class="btn" name="commit" type="submit" value="Create My Account" />
+    </form>-->
+    
+    <!--<%= form_tag("/users", method: "post") do %>
+        <%= label_tag(:username, "Username:") %>
+        <%= text_field_tag(:username) %>
+        <%= label_tag(:email, "Email:") %>
+        <%= text_field_tag(:email) %>
+        <%= label_tag(:password, "Password:") %>
+        <%= password_field_tag(:password) %>
+        <%= submit_tag("Submit") %>
+    <% end %>-->
```

Comment:
Please use `user_path` instead of `/users`

Reply:
Done